.pii_annotations.yml

```diff
@@ -1,7 +1,7 @@
 source_path: ./
 report_path: pii_report
 safelist_path: .annotation_safe_list.yml
-coverage_target: 94.5
```

Comment:
PII annotation coverage has improved even further since my last review! Can you bump this up to 85.3?